bio/genomepy/wrapper.py

```diff
@@ -7,7 +7,7 @@
 from snakemake.shell import shell
 
 # Optional parameters
-provider = snakemake.params.get("provider", "UCSC")
+provider = snakemake.params.get("provider", "ucsc")
```

Comment:
Are the providers always specified in lowercase?
If so, would this make sense?

```suggestion
provider = snakemake.params.get("provider", "ucsc").lower()
```